.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -3,21 +3,31 @@ name: New release template
 about: Template for creating new releases of Durable Functions
 title: ''
 labels: ''
-assignees: cgillum
+assignees: comcmaho, amdeel, davidmrdavid, bachuv
 
 ---
 
-As part of a release, the following items need to be taken care of. For any items that are not applicable, they should be crossed out using the `~~ ~~` markdown syntax with an explanation.
-
-- [ ] Run private scale tests and ensure no regressions
-- [ ] Publish updated versions of [DurableTask.Core](https://www.nuget.org/packages/Microsoft.Azure.DurableTask.Core/) and [DurableTask.AzureStorage](https://www.nuget.org/packages/Microsoft.Azure.DurableTask.AzureStorage/) to nuget.org
+**Prep Release: (assigned to:)**
+_Due: <two-businessdays-before-release>_
 - [ ] Merge all features and fixes into the `master` branch
-- [ ] Update .NET API docs at https://azure.github.io/azure-functions-durable-extension
-- [ ] Publish signed version of [Microsoft.Azure.WebJobs.Extensions.DurableTask](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.DurableTask/) to nuget.org
-- [ ] Update samples to point to the latest nuget packages
+- [ ] Publish updated versions of DurableTask.Core and DurableTask.AzureStorage to app-service MyGet feed
+- [ ] Increment extension version
+- [ ] Publish signed version of Microsoft.Azure.WebJobs.Extensions.DurableTask to app-service MyGet feed
```

Comment:
I realize the following goes over the scope of this PR, but I would love it if this could be automated. There's gotta be some Azure DevOps or GitHub actions way of doing this.

Just a thought, not a blocker!

Reply:
Our release pipeline on Azure DevOps the past two releases has actually done this (and is still wired to do this). I don't plan on changing it back.